pageserver/src/tenant/timeline.rs

```diff
@@ -1284,13 +1284,14 @@ impl Timeline {
             if avg >= Self::VEC_GET_LAYERS_VISITED_WARN_THRESH {
                 use utils::rate_limit::RateLimit;
                 static LOGGED: Lazy<Mutex<RateLimit>> =
-                    Lazy::new(|| Mutex::new(RateLimit::new(Duration::from_secs(10))));
+                    Lazy::new(|| Mutex::new(RateLimit::new(Duration::from_secs(60))));
                 let mut rate_limit = LOGGED.lock().unwrap();
                 rate_limit.call(|| {
                     tracing::info!(
                     tenant_id = %self.tenant_shard_id.tenant_id,
                     shard_id = %self.tenant_shard_id.shard_slug(),
                     timeline_id = %self.timeline_id,
+                    lsn = %lsn,
                     "Vectored read for {} visited {} layers on average per key and {} in total. {}/{} pages were returned",
                     keyspace, avg, layers_visited, results.len(), keyspace.total_raw_size());
                 });
```

Comment:
```suggestion
                      tenant_id = %self.tenant_shard_id.tenant_id,
                      shard_id = %self.tenant_shard_id.shard_slug(),
                      timeline_id = %self.timeline_id,
                      lsn = %lsn,
                      "Vectored read for {} visited {} layers on average per key and {} in total. {}/{} pages were returned",
                      keyspace, avg, layers_visited, results.len(), keyspace.total_raw_size());
                  });
```

annoying rustfmt failure to indent all lines after it indented the first.

Do you think it's possible to log this without already being in a tenant_id/shard_id/timeline_id span? I wouldn't expect that, so we might be able to get rid of those as well as in:

```suggestion
                      lsn = %lsn,
                      "Vectored read for {} visited {} layers on average per key and {} in total. {}/{} pages were returned",
                      keyspace, avg, layers_visited, results.len(), keyspace.total_raw_size());
                  });
```

Reply:
Yeah, tenant and timeline seem to always be present already. Will keep the shard id since that's not always there.

Reply:
Done [32de20e](https://github.com/neondatabase/neon/pull/8263/commits/32de20edd57ac59a2eb203e6fc58cba67c249851)

Reply:
https://github.com/neondatabase/neon/pull/8263/commits/32de20edd57ac59a2eb203e6fc58cba67c249851 looks good, would had probably needed 4 instead of my 2 spaces but :shrug:.